src/bundle.ts

```diff
@@ -95,3 +108,52 @@ export class BuildBundler extends Transform {
         });
   }
 }
+
+export type ResolveFileCallback = (a: string) => void;
+export type RejectFileCallback = (err: Error) => void;
+export type DeferredFileCallbacks = {
+  resolve: ResolveFileCallback; reject: RejectFileCallback;
+};
+
+class StreamLoader implements UrlLoader {
```

Comment:
I decided to change this into a generic FileMapUrlLoader (now a different file and not married to a BuildBundler instance.)